akka-stream/src/main/boilerplate/akka/stream/javadsl/UnzipWith.scala.template

```diff
@@ -0,0 +1,43 @@
+/**
+ * Copyright (C) 2014 Typesafe Inc. <http://www.typesafe.com>
+ */
+package akka.stream.javadsl
+
+import akka.stream._
+import akka.stream.scaladsl
+import akka.japi.function
+import akka.japi.tuple
+import akka.japi
+
+/**
+ * Split one stream into several streams using a splitting function.
+ *
+ * '''Emits when''' all of the outputs stops backpressuring and there is an input element available
+ *
+ * '''Backpressures when''' any of the outputs backpressures
+ *
+ * '''Completes when''' upstream completes
+ *
+ * '''Cancels when''' any downstream cancels
+ */
+object UnzipWith {
+
+  /**
+   * Create a new `UnzipWith` vertex with the specified input type and unzipping-function `f`.
+   *
+   * @param f unzipping-function from the input value to the pair of output values
+   */
+  def createWithPair[In, A, B](f: function.Function[In, japi.Pair[A, B]]):  Graph[FanOutShape2[In, A, B], Unit] =
```

Comment:
Ok, I see that Tuple2 is covered by the below generated `create2` code.
In our Java DSL we have the numbered create starting from the number 3. 
So I'd opt for:

- decide which one we keep we keep `Pair` and `Tuple2`
- make that method `create`
- below code should be generated in range `[3..20]` (since "2" is the normal case)